frontend/src/lib-components/layout/flex-helpers.ts

```diff
@@ -14,6 +15,7 @@ interface FixedSpaceRowProps {
   alignItems?: Property.AlignItems
   fullWidth?: boolean
   maxWidth?: string
+  mobileMargin?: boolean
```

Comment:
I don't think it's good to add one more styling prop just for one use case to a very very widely used component that already has 7 styling props 🤔. It's possible to do this instead in the usage site:

```
const CustomRow = styled(FixedSpaceRow)`
  @media (max-width: ${tabletMin}) {
    margin-right: ${defaultMargins.s};
  }
`
```

Reply:
yeah you're probably right, fixed 👍